runtime/manual/tools/task_runner.md

```diff
@@ -291,12 +300,57 @@ deno run main.ts 2> /dev/null
 deno run main.ts &> /dev/null
 ```
 
-Note that redirecting input and multiple redirects are currently not supported.
+Or redirecting stdout to stderr and vice-versa:
+
+```sh
+# redirect stdout to stderr
+deno run main.ts >&2
+# redirect stderr to stdout
+deno run main.ts 2>&1
+```
+
+Input redirects are also supported:
+
+```sh
+# redirect file.txt to the stdin of gzip
+gzip < file.txt
+```
+
+Note that redirecting multiple redirects is currently not supported.
+
+### Cross-platform shebang
+
+Starting in Deno 1.42, `deno task` will execute scripts that start with
+`#!/usr/bin/env -S` the same way on all platforms.
+
+For example:
+
+```ts title="script.ts"
+#!/usr/bin/env -S deno run
+console.log("Hello there!");
+```
+
+```json title="deno.json"
```

Comment:
I haven't seen that yet! Is that CM or our own syntax extension?

Reply:
Docusaurus https://docusaurus.io/docs/markdown-features/code-blocks#code-title